### PR TITLE
amam/remove_crash_boom

### PR DESCRIPTION
**Pull request by mustofa-binary**
- handle crash/boom in active_symbols api
- handle crash/boom in asset_index api
- handle crash/boom in trading_times api

### CODE REVIEW

File: src/common/util.js [hunk 1]
```diff
@@ -1,3 +1,7 @@
+function isRestrictedSymbol(symbol) {
+  return /^(BOOM|CRASH|STEP).+/i.test(symbol)
+}
```
Comment (negarn) [inline src/common/util.js:3]: why not also write the filtered symbols function here and pass the active symbols to it

File: src/tradingtimes/tradingTimes.es6 [hunk 2]
```diff
@@ -60,6 +66,7 @@ const processData = (markets) => {
       // TODO: comeback and use lodash once 'trade module' changes got merged.
       const market = markets.filter((m) => (m.display_name == marketname))[0];
       const symbols = market && market.submarkets.filter((s) => (s.display_name == submarket_name))[0].instruments;
+      console.log(symbols)
```
Comment (negarn) [inline src/tradingtimes/tradingTimes.es6:69]: missed?

File: src/tradingtimes/tradingTimes.es6 [hunk 3]
```diff
@@ -143,18 +147,26 @@ const initTradingWin = ($html) => {
...
         function changed() {
           const val = $(this).val();
-          header = getObjectMarketSubmarkets(local_storage.get('active_symbols'));
-
-          if (header[val]) submarket_names.update_list(getSortedSubmarkets(Object.keys(header[val])));
+          const new_active_symbols = local_storage.get('active_symbols');
+          const new_filtered_symbols = filterRestrictedSymbols(new_active_symbols)
+          header = getObjectMarketSubmarkets(new_filtered_symbols);
+          
+
```
Comment (negarn) [inline src/tradingtimes/tradingTimes.es6:165]: extra empty line

File: src/tradingtimes/tradingTimes.es6 [hunk 4]
```diff
@@ -143,18 +147,26 @@ const initTradingWin = ($html) => {
 
       /* refresh the table with result of {trading_times:yyyy_mm_dd} from WS */
       const refresh = (data) => {
+         console.log(data[0])
```
Comment (negarn) [inline src/tradingtimes/tradingTimes.es6:150]: console log

File: src/tradingtimes/tradingTimes.es6 [hunk 5]
```diff
@@ -145,16 +149,23 @@ const initTradingWin = ($html) => {
...
-          if (header[val]) submarket_names.update_list(getSortedSubmarkets(Object.keys(header[val])));
+          const new_active_symbols = local_storage.get('active_symbols');
+          const new_filtered_symbols = filterRestrictedSymbols(new_active_symbols)
+          header = getObjectMarketSubmarkets(new_filtered_symbols);
+          
+
+          if (header[val]) {
+             const cumulative_submarkets = Object.keys(header[val])
+             submarket_names.update_list(getSortedSubmarkets(cumulative_submarkets))
+            };
```
Comment (negarn) [inline src/tradingtimes/tradingTimes.es6:168]: indentation for closing bracket, also no semicolon is needed

**Review by negar-binary — APPROVED** [review]

**Pull request merged**